src/DatePicker/DatePicker.test.js

```diff
@@ -33,27 +33,36 @@ describe('<DatePicker />', () => {
 
     test('open/close calendar', () => {
         wrapper = mount(defaultDatePicker);
+        // check to make sure calendar is hidden
         expect(wrapper.state('hidden')).toBeTruthy();
 
+        // click to show calendar
         wrapper.find('input[type="text"]').simulate('click', { type: 'input' });
 
+        // check to make sure calendar is shown
         expect(wrapper.state('hidden')).toBeFalsy();
 
+        // click to hide calendar
```

Comment:
This _seems_ like the wrong comment.  It's simulating a click in the input, which based on the next assertion, should _not_ hide the calendar.

Reply:
@greg-a-smith - thanks for noticing, i just updated the comment